dev-games/ogre/ogre-2.1_pre20180813.ebuild

```diff
@@ -0,0 +1,159 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+CMAKE_REMOVE_MODULES="yes"
+CMAKE_REMOVE_MODULES_LIST="FindFreetype FindDoxygen FindZLIB"
+
+inherit cmake-utils mercurial
+
+DESCRIPTION="Object-oriented Graphics Rendering Engine"
+HOMEPAGE="https://www.ogre3d.org/"
+
+EHG_REPO_URI="https://bitbucket.org/sinbad/ogre"
```

Comment:
Is it not possible with mercurial to point `SRC_URI` to a snapshot download location (passing revision via URI)? Only `trac-mercurial` snapshot ebuild is doing it like here.

Reply:
Good question. Actually I simply looked into the mercurial eclass to see which variables I had to specify a specific commit. If this can be simplified via the SRC_URI, I'd be most happy to change that. (Will research)

I have mercurial installed just for this... :-(

Reply:
Maybe grep through src_uri.*bitbucket for possible examples.